cmd/stcrashreceiver/sentry.go

```diff
@@ -148,22 +148,26 @@ func parseReport(path string, report []byte) (*raven.Packet, error) {
 	if version.commit != "" {
 		pkt.Tags = append(pkt.Tags, raven.Tag{Key: "commit", Value: version.commit})
 	}
+	for _, tag := range version.extra {
+		pkt.Tags = append(pkt.Tags, raven.Tag{Key: "extra", Value: tag})
```

Comment:
I think it's a map serverside, so you want unique keys?

Reply:
Are you sure? I haven't tried but I tried to google it and it seemed like multiple values should be supported. It's a bit of a pain in the ass otherwise, probably better then to send a single joined value

Reply:
The idea is that it would should you the distribution between tags, so I am not sure what the distribution of "extra" means.

I think you can just do `badger=1`, `noupgrade=1`